frontend/src/lib/primevue/useToast.ts

```diff
@@ -1,14 +1,38 @@
+import type { ToastMessageOptions } from 'primevue/toast';
 import { useToast as useToastPrimevue } from 'primevue/usetoast';
 
-import type { ToastMessageOptions } from 'primevue/toast';
+import { ToastTimeout } from '@/utils/constants';
 
 export const useToast = () => {
   const toast = useToastPrimevue();
 
-  const add = (options: ToastMessageOptions = {}) => {
-    const { severity = 'error', summary = 'Unable to load data.', detail = '', life = 5000 } = options;
+  const error = (title: string, msg: string, options: ToastMessageOptions = {}) => {
+    const {
+      severity = 'error',
+      summary = 'Error: ' + title,
```

Comment:
Style: We should be using ES6 Template Literal syntax whenever possible as there is a general performance increase over the older styled string concatenation via +.
Ref: https://stackoverflow.com/a/29083467 (Note that yes, a few years ago, string concatenation was faster than template literal syntax, but modern browser implementations have closed and even surpassed that gap since the original post as seen in the comments).